dev_requirements_pil.txt

```diff
@@ -1,8 +1,11 @@
 --allow-external PIL
 --allow-unverified PIL
+--allow-external pycairo
```

Comment:
if we have pycairo then we do not need to install it here, unless we install a newer version correct?